src/System.Resources.Extensions/ref/System.Resources.Extensions.cs

```diff
@@ -25,13 +25,14 @@ public sealed partial class PreserializedResourceWriter : System.Resources.IReso
         public void AddResource(string name, System.IO.Stream value, bool closeAfterWrite = false) { }
         public void AddResource(string name, object value) { }
         public void AddResource(string name, string value) { }
+        public void AddResource(string name, string typeName, string value) { }
```

Comment:
I don't feel strongly about `AddResource(string, string, string)` versus `AddTypeConverterResource(string, string, string)` writing different things to the .resources file depending on the types involved. From my perspective, that's an implementation detail I don't deeply care about, but I don't mind the rename either.